diff --git a/node_modules/node-html-parser/dist/nodes/html.js b/node_modules/node-html-parser/dist/nodes/html.js
index a166788..36c06ec 100644
--- a/node_modules/node-html-parser/dist/nodes/html.js
+++ b/node_modules/node-html-parser/dist/nodes/html.js
@@ -709,7 +709,7 @@ var HTMLElement = /** @class */ (function (_super) {
             }
             var attrs = {};
             if (this.rawAttrs) {
-                var re = /([a-zA-Z()[\]#@][a-zA-Z0-9-_:()[\]#]*)(?:\s*=\s*((?:'[^']*')|(?:"[^"]*")|\S+))?/g;
+                var re = /([a-zA-Z()[\]#@][a-zA-Z0-9-_:()[\]#\\$]*)(?:\s*=\s*((?:'[^']*')|(?:"[^"]*")|\S+))?/g;
                 var match = void 0;
                 while ((match = re.exec(this.rawAttrs))) {
                     var key = match[1];
